src/bin/cql-stress-cassandra-stress/operation/mixed.rs

```diff
@@ -56,10 +56,16 @@ impl OperationFactory for MixedOperationFactory {
     fn create(&self) -> Box<dyn Operation> {
         let mixed_params = self.settings.command_params.mixed.as_ref().unwrap();
 
-        let write_operation = self.write_operation_factory.create();
-        let counter_write_operation = self.counter_write_operation_factory.create();
-        let read_operation = self.read_operation_factory.create();
-        let counter_read_operation = self.counter_read_operation_factory.create();
+        let write_operation = self.write_operation_factory.as_ref().map(|f| f.create());
+        let counter_write_operation = self
+            .counter_write_operation_factory
+            .as_ref()
+            .map(|f| f.create());
+        let read_operation = self.read_operation_factory.as_ref().map(|f| f.create());
+        let counter_read_operation = self
+            .counter_read_operation_factory
+            .as_ref()
+            .map(|f| f.create());
```

Comment:
We could try to reduce boilerplate by introducing a generic helper function:
```rust
fn produce_operation<Factory: OperationFactory>(factory: &Option<Factory>) -> Factory::Product { 
    factory.as_ref().map(|f| f.create())
}

let write_operation = produce_operation(&self.counter_write_operation_factory);
...
```
I thought up the names; replace them with correct ones. I hope you get the idea.